app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.spec.tsx

```diff
@@ -58,6 +59,12 @@ const DEFAULT_PROPS = {
 };
 
 describe('InPersonLocationStep', () => {
+  const wrapper: ComponentType = ({ children }) => (
+    <InPersonContext.Provider value={{ arcgisSearchEnabled: true }}>
```

Comment:
Why is the ordering of the `InPersonContext.Provider` and the `SWRConfig` reversed in the wrapper as opposed to the original code?

Reply:
Yeah, I don't think this is an issue here, but it seems like values could be overwritten:
"The Provider component accepts a value prop to be passed to consuming components that are descendants of this Provider. One Provider can be connected to many consumers. Providers can be nested to override values deeper within the tree." from [React Docs](https://reactjs.org/docs/context.html#contextprovider).